cartographer/mapping/internal/2d/local_trajectory_builder_2d.cc

```diff
@@ -82,7 +82,7 @@ std::unique_ptr<transform::Rigid2d> LocalTrajectoryBuilder2D::ScanMatch(
         pose_prediction, filtered_gravity_aligned_point_cloud,
```

Comment:
please in line 81. Add const:
`const double score =`

Reply:
Done.